### PR TITLE
Run dev.sh from any path, it should build the agent

### PR DESCRIPTION
**Pull request by nigurr**
Tested on Ubuntu. Should work with all *NIX

### CODE REVIEW

File: src/dev.sh [hunk 1]
```diff
@@ -32,6 +36,7 @@ function failed()
 {
    local error=${1:-Undefined error}
    echo "Failed: $error" >&2
+   cd $CURRENT_DIR
```
Comment (ericsciple) [inline src/dev.sh]: I think it would be a little cleaner to pushd and popd rather than introduce new variables that aren't needed anywhere else.

Comment (nigurr) [inline src/dev.sh]: Done :)

**Comment by ericsciple** [comment]
Wrapping up final testing for a new preview drop. It would be good to hold this for one more day until the release is out the door.

Also it would be good to squash so history isn't messy for anyone trying to investigate meaningful changes.

**Comment by ericsciple** [comment]
@nigurr can you squash before merging to keep history clean (i.e. usable)?

**Comment by nigurr** [comment]
@ericsciple done

**Pull request merged**